capstone/src/main/java/com/google/univiz/api/MapResource.java

```diff
@@ -1,7 +1,15 @@
 package com.google.univiz.api;
 
+import java.util.List;
+
 /**
  * The MapResource interface is responsible for all data management/processing specific for the maps
  * feature of the UniViz webapp.
  */
-public interface MapResource {}
+public interface MapResource {
+  /**
+   * Takes a list of college names and returns a list of MapsData objects to the caller. This object
```

Comment:
Takes a list of CollegeIds?
```suggestion
   * Takes a list of college ids and returns a list of MapsData objects to the caller. This object
```